bin/plugins.json

```diff
@@ -1,60 +1,231 @@
-    {"s3": "opendistro-sql/opendistro_sql", "git": "opendistro-for-elasticsearch/sql"},
-    {"s3": "opendistro-alerting/opendistro_alerting", "git": "opendistro-for-elasticsearch/alerting"},
-    {"s3": "opendistro-job-scheduler/opendistro-job-scheduler", "git": "opendistro-for-elasticsearch/job-scheduler"},
-    {"s3": "opendistro-security/opendistro_security", "git": "opendistro-for-elasticsearch/security"},
-    {"s3": "opendistro-index-management/opendistro_index_management", "git": "opendistro-for-elasticsearch/index-management"},
-    {"s3": "opendistro-anomaly-detection/opendistro-anomaly-detection", "git": "opendistro-for-elasticsearch/anomaly-detection"}
+    {
+      "s3":"opendistro-sql/opendistro_sql",
+      "git":"opendistro-for-elasticsearch/sql",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    },
+    {
+      "s3":"opendistro-alerting/opendistro_alerting",
+      "git":"opendistro-for-elasticsearch/alerting",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    },
+    {
+      "s3":"opendistro-job-scheduler/opendistro-job-scheduler",
+      "git":"opendistro-for-elasticsearch/job-scheduler",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    },
+    {
+      "s3":"opendistro-security/opendistro_security",
+      "git":"opendistro-for-elasticsearch/security",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    },
+    {
+      "s3":"opendistro-index-management/opendistro_index_management",
+      "git":"opendistro-for-elasticsearch/index-management",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    },
+    {
+      "s3":"opendistro-anomaly-detection/opendistro-anomaly-detection",
+      "git":"opendistro-for-elasticsearch/anomaly-detection",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
+    }
   ],
   "rpm":[
-    {"s3": "opendistro-sql/opendistro-sql", "git": "opendistro-for-elasticsearch/sql"},
-    {"s3": "opendistro-alerting/opendistro-alerting", "git": "opendistro-for-elasticsearch/alerting"},
-    {"s3": "opendistro-job-scheduler/opendistro-job-scheduler", "git": "opendistro-for-elasticsearch/job-scheduler"},
-    {"s3": "opendistro-security/opendistro-security", "git": "opendistro-for-elasticsearch/security"},
-    {"s3": "opendistro-performance-analyzer/opendistro-performance-analyzer", "git": "opendistro-for-elasticsearch/performance-analyzer"},
-    {"s3": "opendistro-index-management/opendistro-index-management", "git": "opendistro-for-elasticsearch/index-management"},
-    {"s3": "opendistro-knn/opendistro-knn", "git": "opendistro-for-elasticsearch/k-NN"},
-    {"s3": "opendistro-knnlib/opendistro-knnlib", "git": "opendistro-for-elasticsearch/k-NN"},
-    {"s3": "opendistro-anomaly-detection/opendistro-anomaly-detection", "git": "opendistro-for-elasticsearch/anomaly-detection"}
+    {
+      "s3":"opendistro-sql/opendistro-sql",
+      "git":"opendistro-for-elasticsearch/sql",
+      "curr_version":"1.9.0.0", "next_version":"1.9.1.0"
```

Comment:
I think we should get the versions by checking the release tags. With this way, how should these versions be updated?

Reply:
No, these just like release itself, it is fixed / static once it is out.
I would rather hard code them in json than using release tags.
We only create ODFE ES deb/rpm every ODFE release, so individual plugin release does not matter for build.gradle.

Reply:
but how and when will it be updated?  In the beginning of the release, we can't know which version will be used for each plugin.

Reply:
unless we always use 0.0 for the last 2 digits of plugin version

Reply:
I get it, working on it.

Reply:
Made changes, please check the codes again now.
Thanks.